<img src="https://coursesandconferences.wellcomeconnectingscience.org/wp-content/themes/wcc_courses_and_conferences/dist/assets/svg/logo.svg" width="300" height="50"> 

# Computational Systems Biology for Complex Human Disease 2024

### 21 to 26 April 2024, Wellcome Genome Campus, Hinxton

[Wellcome Connecting Science Course Run Website Link](https://coursesandconferences.wellcomeconnectingscience.org/event/computational-systems-biology-for-complex-human-disease-from-static-to-dynamic-representations-of-disease-mechanisms-20240421/) <br /> 

### Summary

This exciting course will focus on applied, functional analysis and interpretation of disease data using computational modelling tools.

The week-long programme will cover the construction and analysis of both static and dynamic mechanistic networks of disease mechanisms concerning complex human diseases such as cancers, autoimmune, inflammatory diseases and others.

The behaviour of cells is controlled by networks of interacting biomolecules. With biological components represented as “nodes” and the interactions between two components as “edges”, such gene networks can be used to describe the fundamental mechanisms of cellular regulation. These models formalise and integrate large parts of our biological knowledge, and enable novel insights and predictions to be made.

These networks can serve as templates for visualising and analysing “omics” datasets, but they are how ever relatively limited by their static nature. Biological processes are inherently dynamic- they change in response to their environment, stage of development, or mutation. This is determined by how multiple partners interact and work together to activate pathways, and relates to how the order of events within a network may change the functional outcome. The fine-grained relationships between proteins and genes in the cell become particularly important in disease states where the healthy interactions and activities have been subverted.

Executable modelling is a powerful tool to capture the dynamic behaviour of networks, revealing the emergent behaviour of the system under different conditions, through performing in silico simulations and perturbations.

In this course, basic and advanced concepts of mechanistic computational modelling, ranging from pathway models to medical digital twins, will be covered through lectures, keynotes, hands on sessions and project based activities. The participants will be strongly supported to develop from theory to application throughout the week of the course.  The course will also briefly cover the use of omic datasets to either build/ infer models or compare the model’s predictions against gene and protein states found in biological samples. However, applicants should note that this is not a course on data analysis and integration. 

### Target audience

This course is aimed at PhD students, postdocs and clinicians/healthcare professionals who are interested in using systems biology approaches and computational modelling to tackle biological and biomedical problems concerning human disease.

The course is intended to be accessible and beneficial to applicants without advanced bioinformatics skills or experience. However, to fully benefit from the course participants should:

- be familiar with bioinformatics tools, platforms and data resources
- be familiar working in a Linux environment and have a basic knowledge of a programming language (Python, R etc.)

**Numerous free online resources are available for these, including:**

- http://www.ee.surrey.ac.uk/Teaching/Unix/

- https://www.guru99.com/unix-linux-tutorial.html

- https://www.datacamp.com/courses/free-introduction-to-r

- https://www.datacamp.com/courses/intro-to-python-for-data-science

### Programme

Please note that the programme for this course is currently being finalised, and the detailed programme will be available shortly before the beginning of the course.
The course will consist of lectures, discussions, and computational exercises covering as many of the following topics as possible:

**Static Disease Networks (molecular maps, pathway assembly, structural analysis etc.)**

- Overview of databases/ resources
- Different network representations using systems biology graphical notation (SBGN) languages (Process Description, Activity Flow)
- Interactive network visualisation and topological analysis
- Tools: CellDesigner, Cytoscape, REACTOME, MINERVA 

**Dynamic Disease Networks (Stochastic Models, Logical (Boolean)/ Discrete Models, Hybrid models)**

- Network/ Model Curation and sharing
- Disease Boolean Networks 
- Logical/Discrete modelling and simulation of disease networks, and analysis of their dynamical properties
- Stochastic modelling and simulations
- Tools/frameworks/ platforms: BioModels, CaSQ, CellCollective, MaBoSS, PhysiBoss, MetaLo, CoLoMoTo notebook, BioModelAnalyzer, Z3
 
**Workflow**

The week-long course will start from static representations of disease mechanisms in forms of networks, covering aspects of formal representation, available resources, interactive network visualisation, and calculation of topological metrics.

Gradually, with the addition of logical descriptions (automatically or manually) we will pass from static graphs to dynamic networks.

We will explore a number of computational modelling tools that allow for in silico simulations and perturbations in order to understand emerging behaviours of the system under different biological scenarios and move to more in depth dynamical analysis (finding stable states, attractors, performing in silico knockouts, sensitivity analysis etc.). 

The complexity of the tools will also gradually rise (from GUI to program based) following the needs arising from different biological questions.

In parallel, we will showcase how omic data can be used to evaluate a model’s behaviour (set of observations) and guide hypothesis testing (discrete mapping to the system’s attractors). 

Scripts, cheat sheets and other teaching materials will be provided along with instructor support to ensure that all participants are able to follow and progress throughout the week. 

### Learning Outcomes

After completing the course, participants should be able to:

- Understand differences between static and dynamic representations of disease mechanisms
- Understand basic concepts of discrete modelling approaches 
- Retrieve disease networks/ mechanisms from dedicated public repositories and databases and use them as model templates
- Use a range of computational modelling software to develop and analyse discrete computational models
- Use available high and low throughput data to feed and train the models

The participants will benefit from a team that consists of high-level experts in the field, including Professors, and Senior Researchers in fields such as Computational Systems Biology, Computer Science, Bioengineering, Bioinformatics, and other related fields. 

**Course Lead Instructors**      

- [Anna Niariakis](https://orcid.org/0000-0002-9687-7426), Université de Toulouse III - Paul Sabatier, Center of Integrative Biology, France
- [Ben Hall](https://profiles.ucl.ac.uk/33570-benjamin-hall),University College London, UK

**Course Instructors**      

- [Sylvain Soliman](https://lifeware.inria.fr/~soliman/), Inria, France
- [Pedro T. Monteiro](http://pedromonteiro.org/), IST / INESC-ID - University of Lisbon, Portugal
- [Vincent Noël](https://institut-curie.org/personne/vincent-noel), Institut Curie, France
- [David Shorthouse](https://profiles.ucl.ac.uk/82451), University College London, UK
- [Tomas Helikar](https://www.linkedin.com/in/tomashelikar/), University of Nebraska - Lincoln, USA
- [Rahuman S Malik Sheriff](https://www.ebi.ac.uk/people/person/rahuman-sheriff/),European Bioinformatics Institute, UK

**Wellcome Connecting Science Team**

- [Nicola Stevens](https://www.wellcomeconnectingscience.org/person/stevens-nicola/), Events Organiser
- [Martin Aslett](https://coursesandconferences.wellcomeconnectingscience.org/about-us/the-team/), Informatics Manager
- [Vaishnavi Vikas Gangadhar](https://www.wellcomeconnectingscience.org/person/gangadhar-vaishnavi/), Informatics Technical Officer

******

### Citing and Re-using Course Material

The course data are free to reuse and adapt with appropriate attribution. All course data in these repositories are licensed under the <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</a>. <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons Licence" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br /> 

Each course landing page is assigned a DOI via Zenodo, providing a stable and citable reference. These DOIs can be found on the respective course landing pages and can be included in CVs or research publications, offering a professional record of the course contributions.

### Interested in attending a course?

Take a look at what courses are coming up at [Wellcome Connecting Science Courses & Conference Website](https://coursesandconferences.wellcomeconnectingscience.org/our-events/).

---

For more information or queries, feel free to contact us via the [Wellcome Connecting Science website](https://coursesandconferences.wellcomeconnectingscience.org).<br /> 
Find us on socials [Wellcome Connecting Science Linktr](https://linktr.ee/eventswcs)

---

[Wellcome Connecting Science GitHub Home Page](https://github.com/WCSCourses) <br /> 
